test/integration/adoptable_dog_show_test.rb

```diff
@@ -3,7 +3,7 @@
 class AdoptableDogShowTest < ActionDispatch::IntegrationTest
 
   setup do
-    @dog_id = dogs(:dog_three).id
+    @dog_id = dogs(:one).id
```

Comment:
yes, I like this rename, it's more streamlined when calling the fixture this way.